application/cms/form_fields.py

```diff
@@ -205,6 +214,7 @@ def __init__(self, label=None, validators=None, hint=None, extended_hint=None, s
 
         super().__init__(label, validators, **kwargs)
         self.hint = hint
+        self.character_count_limit = character_count_limit
```

Comment:
I know that I technically added this functionality, but I feel like we definitely want a test for this new feature. If you don't feel comfortable testing it, I can make some time to write one today.

Reply:
Any thoughts on the best way to test this? A functional test checking that the outputted HTML has the extra bits? Unit test? I'm not sure we need to test the character counting functionality with with full Selenium stack, given that GDS have already written tests around it?

Reply:
Yep, maybe just that it provides the extra text on the page beneath the box. That's probably sufficient for now.